RSS Reader/RSS Reader/View Controllers/Topics list/View /RSSRTopicsListViewController.m

```diff
@@ -132,7 +157,8 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
     RSSRTopicTableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:kReuseIdentifier forIndexPath:indexPath];
-    [cell configureWithItem:[self.presenter topics][indexPath.row]];
+    [cell configureWithItem:[self.presenter topics][indexPath.row]
+                   delegate:self];
```

Comment:
I would prefer to use blocks instead of delegate callback. No need to fix it, just keep it in mind